lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -97,6 +113,10 @@
 # The certificates are not necessary for testing, wherein the PUT request is mocked.
 ssl_context = None
 
+# Use mock API Key in env variables for testing, else set to None
+# Outside of testing, COMP_AND_PEN_OPT_IN_API_KEY is defined by its paramater store value
+COMP_AND_PEN_OPT_IN_API_KEY = os.getenv('COMP_AND_PEN_OPT_IN_API_KEY', None)
```

Comment:
NOT IDEAL but this will corrected in an new user story to refactor testing.  In testing, this constant should be populated by a mocked called to the `ssm` resource. However, logic around test ENV and integration testing make this difficult.